plugins/services/src/js/utils/ServiceTreeUtil.js

```diff
@@ -0,0 +1,13 @@
+import ServiceTree from "../structs/ServiceTree";
+
+const ServiceTreeUtil = {
+  isGroupWithServices(service) {
+    if (service instanceof ServiceTree && service.getListCount() > 0) {
```

Comment:
not big deal but can also be written as `return service instanceof ServiceTree && service.getListCount() > 0`